scipy/interpolate/tests/test_interpnd.py

```diff
@@ -166,6 +167,43 @@ def test_pickle(self):
 
         assert_almost_equal(ip(0.5, 0.5), ip2(0.5, 0.5))
 
+    def test_threading(self):
```

Comment:
Would be nice to add a comment that the test originates from https://github.com/scipy/scipy/issues/8856